Specs/DataSources/CzmlDataSourceSpec.js

```diff
@@ -270,11 +266,9 @@ defineSuite([
     });
 
     it('process loads data on top of existing', function() {
-        waitsFor(function() {
+        return pollToPromise(function() {
             return defined(simple) && defined(vehicle);
```

Comment:
Assuming that the changes to `beforeAll` above work properly (that is, assuming beforeAll doesn't complete until the returned promise has resolved), I think this `pollToPromise` here (and similar throughout) is unnecessary.